usecases/auth/authorization/rbac/manager.go

```diff
@@ -96,6 +96,10 @@ func (m *manager) RemovePermissions(roleName string, permissions []*authorizatio
 	return m.casbin.InvalidateCache()
 }
 
+func (m *manager) HasPermission(role string, permission *authorization.Policy) (bool, error) {
+	return m.casbin.Enforce(conv.PrefixRoleName(role), permission.Resource, permission.Verb)
```

Comment:
**issue (blocking):** casbin has method `HasNamedPolicy()` https://casbin.org/docs/management-api/#hasnamedpolicy for this, `Enforce()` will check if the user has the requested permission